tools/godotcpp.py

```diff
@@ -467,8 +467,6 @@ def generate(env):
         # DEBUG_ENABLED enables debugging *features* and debug-only code, which is intended
         # to give *users* extra debugging information for their game development.
         env.Append(CPPDEFINES=["DEBUG_ENABLED"])
-        # In upstream Godot this is added in typedefs.h when DEBUG_ENABLED is set.
```

Comment:
What's the reason for moving this? Or rather, I guess was there a reason not to have it in `typedefs` for godot-cpp?

Reply:
`DEBUG_METHODS_ENABLED` is used only by `binder_common.hpp` - are we sure that `defs.hpp` will be included there?

Reply:
I've explicitly added the `defs.hpp` include to `binder_common.hpp` just in case.